library/src/interactor_impl.cxx

```diff
@@ -817,7 +854,18 @@ bool interactor_impl::triggerCommand(std::string_view command)
     return true;
   }
 
-  const std::string& action = tokens[0];
+  std::string action = tokens[0];
+
+  // Resolve Alias
+  auto aliasIt = AliasMap.find(action);
+  if (aliasIt != AliasMap.end())
+  {
+    std::vector<std::string> aliasTokens = utils::tokenize(aliasIt->second);
+    tokens.erase(tokens.begin());
+    tokens.insert(tokens.begin(), aliasTokens.begin(), aliasTokens.end());
+    action = tokens[0];
+  }
```

Comment:
there is an even simpler way to implement this, just look for the command in the aliasMap at the very begining of the triggerCommand method.